### PR TITLE
Update version to 1.0.0 | Add env vars for Oauth2

### PR DESCRIPTION
**Pull request by dimdimitros**

### CODE REVIEW

**Review by cjd1884 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -42,11 +42,17 @@ Finally, you need to pass the `API_KEY` and `API_URL` as `resValues` inside your
 ```kotlin
 resValue("string", "antifraud_sdk_key", YOUR_API_KEY)
 resValue("string", "antifraud_sdk_api_url", YOUR_API_URL)
+resValue("string", "antifraud_sdk_client_id", YOUR_ANTIFRAUD_CLIENT_ID)
```
Comment (cjd1884) [inline README.md]: @dimdimitros `YOUR_ANTIFRAUD_CLIENT_ID` ->  `YOUR_CLIENT_ID`

File: README.md [hunk 2]
```diff
@@ -42,11 +42,17 @@ Finally, you need to pass the `API_KEY` and `API_URL` as `resValues` inside your
...
+resValue("string", "antifraud_sdk_client_secret", YOUR_ANTIFRAUD_CLIENT_SECRET)
```
Comment (cjd1884) [inline README.md]: @dimdimitros `YOUR_ANTIFRAUD_CLIENT_SECRET` ->  `YOUR_CLIENT_SECRET`

File: README.md [hunk 3]
```diff
@@ -42,11 +42,17 @@ Finally, you need to pass the `API_KEY` and `API_URL` as `resValues` inside your
...
+resValue("string", "antifraud_sdk_oauth2_token_url", YOUR_ANTIFRAUD_OAUTH2_TOKEN_URL)
```
Comment (cjd1884) [inline README.md]: @dimdimitros `YOUR_ANTIFRAUD_OAUTH2_TOKEN_URL ` ->  `OAUTH2_TOKEN_URL`

File: README.md [hunk 4]
```diff
@@ -42,11 +42,17 @@ Finally, you need to pass the `API_KEY` and `API_URL` as `resValues` inside your
 ```kotlin
 resValue("string", "antifraud_sdk_key", YOUR_API_KEY)
 resValue("string", "antifraud_sdk_api_url", YOUR_API_URL)
```
Comment (cjd1884) [inline README.md]: @dimdimitros `YOUR_API_URL` -> `API_URL `

File: README.md [hunk 5]
```diff
@@ -41,12 +41,18 @@ dependencies {
...
+resValue("string", "antifraud_sdk_client_secret", YOUR_CLIENT_SECRET)
+resValue("string", "antifraud_sdk_oauth2_token_url", OAUTH2_TOKEN_URL)
 ```
 or via your app's `strings.xml` file
 ```xml
 <string name="antifraud_sdk_key">YOUR_API_KEY</string>
 <string name="antifraud_sdk_api_url"> YOUR_API_URL</string>
+<string name="antifraud_sdk_client_id">YOUR_ANTIFRAUD_CLIENT_ID</string>
+<string name="antifraud_sdk_client_secret"> YOUR_ANTIFRAUD_CLIENT_SECRET</string>
+<string name="antifraud_sdk_oauth2_token_url">YOUR_ANTIFRAUD_OAUTH2_TOKEN_URL</string>
```
Comment (cjd1884) [inline README.md]: `YOUR_ANTIFRAUD_OAUTH2_TOKEN_URL` -> `ANTIFRAUD_OAUTH2_TOKEN_URL`

File: README.md [hunk 6]
```diff
@@ -41,12 +41,18 @@ dependencies {
...
-resValue("string", "antifraud_sdk_api_url", YOUR_API_URL)
+resValue("string", "antifraud_sdk_api_url", API_URL)
+resValue("string", "antifraud_sdk_client_id", YOUR_CLIENT_ID)
+resValue("string", "antifraud_sdk_client_secret", YOUR_CLIENT_SECRET)
+resValue("string", "antifraud_sdk_oauth2_token_url", OAUTH2_TOKEN_URL)
 ```
 or via your app's `strings.xml` file
 ```xml
 <string name="antifraud_sdk_key">YOUR_API_KEY</string>
 <string name="antifraud_sdk_api_url"> YOUR_API_URL</string>
```
Comment (cjd1884) [inline README.md]: `YOUR_API_URL ` -> `API_URL `

**Review by cjd1884 — COMMENTED** [review]
@dimdimitros Do the minor renames. Other than that, looks good. 👍

**Comment by dimdimitros** [comment]
> @dimdimitros Do the minor renames. Other than that, looks good. 👍

I will keep the same naming convention to both resValues and strings.xml 
```
resValue("string", "antifraud_sdk_key", YOUR_API_KEY)
resValue("string", "antifraud_sdk_api_url", API_URL)
resValue("string", "antifraud_sdk_client_id", YOUR_CLIENT_ID)
resValue("string", "antifraud_sdk_client_secret", YOUR_CLIENT_SECRET)
resValue("string", "antifraud_sdk_oauth2_token_url", OAUTH2_TOKEN_URL)

and

<string name="antifraud_sdk_key">YOUR_API_KEY</string>
<string name="antifraud_sdk_api_url"> API_URL</string>
<string name="antifraud_sdk_client_id">YOUR_CLIENT_ID</string>
<string name="antifraud_sdk_client_secret"> YOUR_CLIENT_SECRET</string>
<string name="antifraud_sdk_oauth2_token_url">OAUTH2_TOKEN_URL</string>```

**Pull request merged**